### PR TITLE
Improve docs - README

### PR DESCRIPTION
**Pull request by mvorisek**
Based on https://github.com/symfony/symfony/blob/1b4ab8108542daddd6ca96caf38b464ff3cc459a/src/Symfony/Component/Finder/Finder.php#L317

### CODE REVIEW

File: README.rst [hunk 1]
```diff
@@ -2022,7 +2022,7 @@ The example below will add two rules to the default list of PSR2 set rules:
         ->setFinder($finder)
     ;
 
-**NOTE**: ``exclude`` will work only for directories, so if you need to exclude file, try ``notPath``.
+**NOTE**: ``exclude`` will work only for directories, so if you need to exclude file, try ``notPath``. Both ``exclude`` and ``notPath`` methods accepts only relative paths to the ones defined with the ``in`` method.
```
Comment (kubawerlos) [inline README.rst]:
```suggestion
**NOTE**: ``exclude`` will work only for directories, so if you need to exclude file, try ``notPath``. Both ``exclude`` and ``notPath`` methods accept only relative paths to the ones defined with the ``in`` method.
```

Comment (mvorisek) [reply README.rst]: Plural - so it should be accept, should not?

**Comment by mvorisek** [comment]
> 
> 
> Also tests are failing - see the README information at https://github.com/FriendsOfPHP/PHP-CS-Fixer/blob/v2.16.0/CONTRIBUTING.md

What should I do? This is only improval of README, not code, I do not except any CI tests on this.

**Comment by kubawerlos** [comment]
@mvorisek check the message from the failing test ;)

>  I do not except any CI tests on this

How do you think README is up to date with all the fixers in code?

**Comment by mvorisek** [comment]
> 
> 
> @mvorisek check the message from the failing test ;)
> 
> > I do not except any CI tests on this
> 
> How do you think README is up to date with all the fixers in code?

Fixed.

**Comment by keradus** [comment]
Thank you @mvorisek.

**Comment by keradus** [comment]
(merged to 2.15 instead of master)

**Pull request merged**